compiler/gopherjspkg/fs.go

```diff
@@ -23,6 +24,12 @@ var FS = filter.Keep(
 )
 
 func importPathToDir(importPath string) string {
+	for _, src := range build.Default.SrcDirs() {
+		dir := filepath.Join(src, importPath)
+		if _, err := os.Stat(dir); err == nil {
+			return dir
+		}
+	}
```

Comment:
Why is this needed? I'd expect build.Import() to find packages in standard locations. Same below.